Workbooks/Workloads/Create log alert/Create log alert.workbook

```diff
@@ -78,7 +78,7 @@
             "label": "Alert name",
             "type": 1,
             "isRequired": true,
-            "query": "Resources\r\n| take 1\r\n| project id = extract(@'/subscriptions/.+/resourcegroups/.+/providers/microsoft.operationalinsights/workspaces/.+/alertname/(.+)/threshold/.+/template/.+', 1, tolower('{WorkspaceAndThrehsold}'))\r\n| project value = id, label = id, selected = true",
+            "query": "Resources\r\n| take 1\r\n| project id = extract(@'/subscriptions/.+/resourcegroups/.+/providers/microsoft.operationalinsights/workspaces/.+/alertname/(.+)/threshold/.+/template/.+', 1, '{WorkspaceAndThrehsold}')\r\n| project value = id, label = id, selected = true",

```

Comment:
I don’t think this is a safe match operation. The resource id can have different casing that can fail this regex match (e.g. resourceGroup instead of resourcegroup. Find the regex option that ignores case if you want to preserve casing of names (not sure if KQL supports it).

Reply:
Changed to case insensitive (?i) - we might consider this elsewhere instead of lower-casing